templates/gallery/gallery-inner.amp.html

```diff
@@ -0,0 +1,207 @@
+<!doctype html>
+
+<!---
+Copyright 2017 The AMP Start Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+{{#gallery}}
+<html ⚡>
+  {{> ../../utils/amp-page-head.snip.html}}
+  <body>
+    <main id="content" class="gallery-base-content relative fit" role="main">
+      <svg
+        class="gallery-inner-background-1"
+        xmlns="http://www.w3.org/2000/svg"
+        viewBox="0 0 388 809"
+      >
+        <path
+          d="
+            M7,142.5 C7,245 124.5,286 124.5,373 C124.5,460 47.5,579.367391
+            47.5,679 C47.5,778.632609 140.5,802 198,802 C255.5,802 338,803.5
+            338,671.5 C338,539.5 321.5,560.5 321.5,444.5 C321.5,328.5 381,274
+            381,134 C381,-6 268.5,8 203,8 C137.5,8 7,40 7,142.5 Z
+          "
+          stroke="none"
+          fill="#F5F6FD"
+        ></path>
+      </svg>
+      <svg
+        class="gallery-inner-background-2"
+        xmlns="http://www.w3.org/2000/svg"
+        viewBox="0 0 1167 599"
+      >
+        <path
+          d="
+            M8,359.5 C8,503 64.5,538.5 185.5,538.5 C306.5,538.5 335.554256,508.5
+            471.5,508.5 C607.445744,508.5 737,594 879.5,594 C1022,594 1159,543.5
+            1159,299.5 C1159,55.5 1062.73379,5 902,5 C741.266207,5 689,220
+            568,220 C447,220 234.617034,147.5 143,147.5 C51.3829655,147.5 8,216
+            8,359.5 Z
+          "
+          stroke="none"
+          fill="#F5F6FD"
+        ></path>
+      </svg>
+      {{> ./partials/hero.snip.html}}
+      <section class="gallery-base-section gallery-base-rich-text">
+        <div class="gallery-base-container mx-auto">
+          <div class="clearfix">
```

Comment:
you will not need to use clearfix if you use flexboxes - which is super easy in bass, any reason why we are using floats (.col)